l1-contracts/scripts/token-migration.ts

```diff
@@ -218,3 +221,40 @@ async function prepareGovernanceTokenMigrationCall(
     executeCalldata,
   };
 }
+
+const provider = web3Provider();
+
+export async function transferTokensOnForkedNetwork(deployer: Deployer) {
+  const tokenList = ["0x5A520e593F89c908cd2bc27D928bc75913C55C42"];
```

Comment:
please add a comment on how this function is meant to be used and why does it have hardcoded tokenList.